src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -211,7 +217,9 @@ class Diagram extends Component<Props> {
     if (
       !prevState.editingNodeItem &&
       this.props.currentFlowNode?.isNew &&
-      (this.props.currentFlowNode?.type === 'say_something' || this.props.currentFlowNode?.type === 'trigger')
+      (this.props.currentFlowNode?.type === 'say_something' ||
+        this.props.currentFlowNode?.type === 'trigger' ||
+        this.props.currentFlowNode?.type === 'prompt')
```

Comment:
`['say_something', 'trigger', 'prompt'].includes(this.props.currentFlowNode?.type)`